load-test/config/default.json

```diff
@@ -0,0 +1,20 @@
+{
+  "auth": {
+    "clientId": "chefs-load-test",
+    "host": "https://dev.oidc.gov.bc.ca",
+    "realm": "cp1qly2d",
+    "password": "",
+    "username": "chefs-load-test"
+  },
+  "api": {
+    "apiPath": "api/v1",
+    "basePath" : "/app",
+    "host": "https://chefs-dev.pathfinder.gov.bc.ca"
+  },
+  "submissions": {
+    "count": 1000,
```

Comment:
woah! this all looks awesome.. pshhh.. 
I was going to try running it locally at some point.. then was concerned i would crash something.
maybe this count should have a lower default? you mentioned 5 above

Reply:
you can lower it in a local.json or by environment variable. you need to set the password anyway.